js/views.js

```diff
@@ -0,0 +1,75 @@
+// Todo View
+App.Views.Todo = Backbone.View.extend({
+    template: _.template(
+        '<div class="todo-item">' +
+        '<span class="<%= isComplete ? \'complete\' : \'\' %>"><input type="checkbox" <%= isComplete ? \'checked\' : \'\' %> />' +
+        '<%= description %></span>' +
+        '<i class="material-icons clear">clear</i>' +
+        '</div>'
+    ),
+
+    initialize: function () {
+        this.listenTo(this.model, 'change', this.render);
+        this.listenTo(this.model, 'destroy', this.remove);
+    },
+
+    events: {
+        'change input': 'toggleComplete',
+        'click i': 'deleteTodo'
+    },
+
+    toggleComplete: function () {
+        this.model.toggleComplete();
+    },
+
+    deleteTodo: function () {
+        this.model.destroy();
+    },
+
+    render: function () {
+        this.$el.html(this.template(this.model.attributes));
+    }
+});
+
+// Create Todo View
+App.Views.TodoForm = Backbone.View.extend({
+    template: _.template(
+        '<form class="todo-form"><input type="text" id="todoText" /><button>Save</button></form>'
+    ),
+
+    events: {
+        submit: 'addTodo'
+    },
+
+    addTodo: function (e) {
+        e.preventDefault();
+        let inputEl = $('#todoText')[0];
```

Comment:
Prefer using `this.$('#todoText')`. This ensures that when you're selecting for stuff you're scoping your selection to the specific view (instead of the page as a whole).

Also along those same lines: pretty much never use ids (#). The speed increase is negligible and it makes your components not reusable. Classes FTW.